infra/base-images/base-runner/coverage

```diff
@@ -127,9 +129,13 @@ llvm-cov show -format=html -output-dir=$REPORT_DIR \
 # Export coverage summary in JSON format.
 llvm-cov export -summary-only $LLVM_COV_ARGS > $SUMMARY_FILE
 
+# Post process HTML report.
+coverage_helper -v post_process -output-dir=$REPORT_DIR -src-root-dir=$OUT \
```

Comment:
nit: i think -src-root-dir looks better first, followed by output ones like -output-dir -summary-file

Reply:
Yeah, good point, src root and summary are inputs, so I'll pass them first. Thanks!